### ISSUE TITLE
community guidelines missing

### ISSUE THREAD

**Issue by cmaimone** [hknd23/idcempy#16]
From the JOSS review criteria:

Community guidelines

There should be clear guidelines for third-parties wishing to:

    Contribute to the software
    Report issues or problems with the software
    Seek support

This can be done via a contributing file in GitHub - see https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors

**Comment by hknd23** [comment]
We have added the contributing file to the package’s GitHub repository as well as a section for community guidelines in the documentation. Issues can be raised for questions and bug reports, and contributors can contact the authors through email for suggestions.

**Comment by cmaimone** [comment]
I'm not seeing the contributing file.  I see a brief Contributions section in the readthedocs documentation and readme.md file, but it doesn't tell people how to submit issues or PRs, report bugs, etc. You'll need to go beyond just telling people to email you for this to really be functional open source software.

Take a look at some of the other published packages on JOSS.  Here's a recent one I grabbed from the website: https://github.com/DrMarc/slab

**Comment by hknd23** [comment]
We apologize for not thoroughly addressing your point about the contributing file (in other words, contribution guide) in our previous round of revisions. Thank you for providing the link to the contributing file in a publicly available Github repo. This is an extremely helpful example. We have added a detailed contribution guide to our repo using both the template from your suggested link and other available templates for contributing files as a reference point. Our revised and updated contribution guide is included in the readthedocs documentation and readme.md file available at https://github.com/hknd23/idcempy/blob/main/CONTRIBUTING.md. This link provides details about the contents included in the contribution guide associated with our IDCeMPy package. Stated briefly, however, the updated contribution guide is divided into the following five categories: Code of Conduct, Bug Reports, Features Requests, Pull Requests (PRs), License, Getting Help. Detailed information and guidance to potential contributors on how to report bugs, provide PRs as well as Features requests and so on is provided in the relevant categories mentioned above. In addition, we have added issue templates for bug reports and feature requests to facilitate the processes.
